back-end/src/Person.java

```diff
@@ -50,4 +51,10 @@ public LocalDate getBirthday() {
     public void introduce() {
         System.out.println("Hello I am person");
     }
+
+    public void displayAllInformation() {
+        System.out.println("Họ và tên: " + this.getLastName() + " " + this.getFirstName());
```

Comment:
Print first line: "Thông tin chi tiếc của ..." 
... là Học sinh hoặc Giáo Viên.

Print secondline: "Mã: ..."
.. là mã sinh viên hoặc là mã giáo viên 


Please write the function getType() from Person and override in Student and Teach.

Reply:
In Person class:
System.out.println("Thông tin chi tiếc của" + getType()");

public String getType() {
return "Người";
}